.changeset/good-years-bake.md

```diff
@@ -0,0 +1,5 @@
+---
+'@finos/legend-studio': patch
+---
+
+A RawLambda resolver is added to resolve element paths inside lambdas that leverage imports. The resolver was added since the SDLC server does not currently support the use of sections (element containing imports). Currently, we resolve element paths everywhere in the app except in the lambdas since lambdas are kept in the format they are saved in the SDLC server. This resolver allows the graph to be compiled without the need to save the section elements.
```

Comment:
You're actually going the opposite direction this time 😭 But I love the details. I think we keep this as the PR description and simplify this down. This is a changelog so people want faster read.